tests/NewCypress/cypress/e2e/regression/pagination.cy.ts

```diff
@@ -0,0 +1,82 @@
+import paginationPage from "../../pages/paginationPage";
+
+describe('Pagination Tests', () => {
+
+
+    beforeEach(() => {
+        cy.login()
+        cy.visit('/search?keywords=west')
```

Comment:
We're only testing one search result which currently has three result pages. It would be good to test a couple of scenarios with different numbers of page results, especially as the ui behaves differently depending on whether there are 1, 2, 3 or more pages of results

Reply:
Now added another test that paginates through a search page with a large number of results